File: js/antiqueStoreNames.js
function nameGen(){
	var nm1 = ["A New View","About Time","Age of Vintage","Aged Goodness","Anecdotes","Another Man's Treasures","Another's Treasures","Antique Appeal","Antique Cache","Antique Cuties","Antique Hut","Antique Trove","Antiques and Artifacts","Antiquities","Authentic Antiquities","Beautiful Salvage","Blasts from the Past","Bought Again","Bygones","Checkered Pasts","Cozy Collectibles","Dinky and Dainty","Discoveries","Drab to Fab","Echoes of the Past","Established Goods","Focus of the Past","Focus on the Past","Forget Me Nots","Forgotten Furnishings","Fortunes in Time","Found in Time","From Oblivion","From Time Immemorial","Frozen in Time","General Goods","Good As New","Good Goods","Good Ol' Days","Hodge Podge Lodge","Honest Heirlooms","In-Of-Date","Joys For Forever","Junk Deluxe","Just in Time","Knick-knack Paddywhack Shack","Legends of Their Time","Little Collectibles","Little House of Trinkets","Live in the Past","Living Memories","Long Time No See","Lost and Found in Time","Matters of Time","Memory Lane","Miracles of the Past","Mod Life","Modern Memories","Modern Vintage","Needful Things","Odd Types","Old Gold","Old Roots","Old Stuff","Old and Bold","Oldies and Goldies","Once upon a time","One Time or Another","Out of the Attic","Pandora's Box","Paragon Prizes","Pass On the Past","Pass The Time","Past Caring","Past Meets Present","Past On","Past Over","Past Passed On","Past Repast","Pockets of Time","Precious Past","Precious Things","Presents for the Present","Presents of the Past","Preserved","Primes of the Past","Rags and Riches","Recent Memories","Recollect Them All","Recollectibles","Relics and Rarities","Relics and Riches","Relics of Time","Reliquary","Remains to be Seen","Remember","Remember This","Remember, Remember","Renewed Life","Renewed Memories","Retro Relics","Revered Relics","Revival","Salvage Garden","Secrets of the Past","Shared Memories","Stuff and Things","Sweet Memories","Taken for Granted","Tales Resold","Tales of the Past","Tales Retold","Tales Untold","Tests of Time","The Antiki-Wiki","The Antique Market","The Salvage Beast","The Shuffle of Things","The Time is Ripe","The Treasure Chest","The Treasure Trove","Things Past","Things of the Past","Thingymabobs","Time Honored","Time Not Forgotten","Time Recovered","Time of One's Life","Timeless Treasures","Timeless Trinkets","Times Remembered","Top Drawer Antiques","Treasures and Trinkets","Treasures of Time","Trinkets and Traditions","Trinkets of Time","Twosided","Utter Clutter","Vagabond Vendor","Vestiges of the Past","Vintage","Vintage Baby","Vintage Treasures","Vintage Vogue","Vintage Wares","Warm Wares","Wayward Wealths","Wealth of Time","Well of a Time","Whispers of the Past","Wonders of the Past"];
	var br = "";
	var element = document.createElement("div");
	element.setAttribute("id", "result");
	
	for(i = 0; i < 10; i++){
		rnd = Math.floor(Math.random() * nm1.length);
		names = nm1[rnd];
		nm1.splice(rnd, 1);
		br = document.createElement('br');	
		element.appendChild(document.createTextNode(names));
		element.appendChild(br);
	}
	if(document.getElementById("result")){
		document.getElementById("placeholder").removeChild(document.getElementById("result"));
	}		
	document.getElementById("placeholder").appendChild(element);
}

File: generators/antiqueStoreNames.py
__all__ = ['antiqueStoreNames']

# Don't look below, you will not understand this Python code :) I don't.

from js2py.pyjs import *
# setting scope
var = Scope( JS_BUILTINS )
set_global_object(var)

# Code follows:
var.registers(['nameGen'])
@Js
def PyJsHoisted_nameGen_(this, arguments, var=var):
    var = Scope({'arguments':arguments, 'this':this}, var)
    var.registers(['nm1', 'result'])
    var.put('nm1', Js([Js('A New View'), Js('About Time'), Js('Age of Vintage'), Js('Aged Goodness'), Js('Anecdotes'), Js("Another Man's Treasures"), Js("Another's Treasures"), Js('Antique Appeal'), Js('Antique Cache'), Js('Antique Cuties'), Js('Antique Hut'), Js('Antique Trove'), Js('Antiques and Artifacts'), Js('Antiquities'), Js('Authentic Antiquities'), Js('Beautiful Salvage'), Js('Blasts from the Past'), Js('Bought Again'), Js('Bygones'), Js('Checkered Pasts'), Js('Cozy Collectibles'), Js('Dinky and Dainty'), Js('Discoveries'), Js('Drab to Fab'), Js('Echoes of the Past'), Js('Established Goods'), Js('Focus of the Past'), Js('Focus on the Past'), Js('Forget Me Nots'), Js('Forgotten Furnishings'), Js('Fortunes in Time'), Js('Found in Time'), Js('From Oblivion'), Js('From Time Immemorial'), Js('Frozen in Time'), Js('General Goods'), Js('Good As New'), Js('Good Goods'), Js("Good Ol' Days"), Js('Hodge Podge Lodge'), Js('Honest Heirlooms'), Js('In-Of-Date'), Js('Joys For Forever'), Js('Junk Deluxe'), Js('Just in Time'), Js('Knick-knack Paddywhack Shack'), Js('Legends of Their Time'), Js('Little Collectibles'), Js('Little House of Trinkets'), Js('Live in the Past'), Js('Living Memories'), Js('Long Time No See'), Js('Lost and Found in Time'), Js('Matters of Time'), Js('Memory Lane'), Js('Miracles of the Past'), Js('Mod Life'), Js('Modern Memories'), Js('Modern Vintage'), Js('Needful Things'), Js('Odd Types'), Js('Old Gold'), Js('Old Roots'), Js('Old Stuff'), Js('Old and Bold'), Js('Oldies and Goldies'), Js('Once upon a time'), Js('One Time or Another'), Js('Out of the Attic'), Js("Pandora's Box"), Js('Paragon Prizes'), Js('Pass On the Past'), Js('Pass The Time'), Js('Past Caring'), Js('Past Meets Present'), Js('Past On'), Js('Past Over'), Js('Past Passed On'), Js('Past Repast'), Js('Pockets of Time'), Js('Precious Past'), Js('Precious Things'), Js('Presents for the Present'), Js('Presents of the Past'), Js('Preserved'), Js('Primes of the Past'), Js('Rags and Riches'), Js('Recent Memories'), Js('Recollect Them All'), Js('Recollectibles'), Js('Relics and Rarities'), Js('Relics and Riches'), Js('Relics of Time'), Js('Reliquary'), Js('Remains to be Seen'), Js('Remember'), Js('Remember This'), Js('Remember, Remember'), Js('Renewed Life'), Js('Renewed Memories'), Js('Retro Relics'), Js('Revered Relics'), Js('Revival'), Js('Salvage Garden'), Js('Secrets of the Past'), Js('Shared Memories'), Js('Stuff and Things'), Js('Sweet Memories'), Js('Taken for Granted'), Js('Tales Resold'), Js('Tales of the Past'), Js('Tales Retold'), Js('Tales Untold'), Js('Tests of Time'), Js('The Antiki-Wiki'), Js('The Antique Market'), Js('The Salvage Beast'), Js('The Shuffle of Things'), Js('The Time is Ripe'), Js('The Treasure Chest'), Js('The Treasure Trove'), Js('Things Past'), Js('Things of the Past'), Js('Thingymabobs'), Js('Time Honored'), Js('Time Not Forgotten'), Js('Time Recovered'), Js("Time of One's Life"), Js('Timeless Treasures'), Js('Timeless Trinkets'), Js('Times Remembered'), Js('Top Drawer Antiques'), Js('Treasures and Trinkets'), Js('Treasures of Time'), Js('Trinkets and Traditions'), Js('Trinkets of Time'), Js('Twosided'), Js('Utter Clutter'), Js('Vagabond Vendor'), Js('Vestiges of the Past'), Js('Vintage'), Js('Vintage Baby'), Js('Vintage Treasures'), Js('Vintage Vogue'), Js('Vintage Wares'), Js('Warm Wares'), Js('Wayward Wealths'), Js('Wealth of Time'), Js('Well of a Time'), Js('Whispers of the Past'), Js('Wonders of the Past')]))
    var.put('result', Js([]))
    #for JS loop
    var.put('i', Js(0.0))
    while (var.get('i')<Js(10.0)):
        try:
            var.put('rnd', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm1').get('length'))))
            var.put('names', var.get('nm1').get(var.get('rnd')))
            var.get('nm1').callprop('splice', var.get('rnd'), Js(1.0))
            var.get('result').callprop('push', var.get('names'))
        finally:
                (var.put('i',Js(var.get('i').to_number())+Js(1))-Js(1))
    pass
    return var.get('result')
PyJsHoisted_nameGen_.func_name = 'nameGen'
var.put('nameGen', PyJsHoisted_nameGen_)
pass
pass


# Add lib to the module scope
antiqueStoreNames = var.to_python()